devops/openshift/create_nice_installation.rst

```diff
@@ -13,6 +13,10 @@ Step by Step Instructions
 
     .. figure:: create_nice_installation/create_project.png
         :scale: 60%
+    
+    #. The name of your account
+
```

Comment:
shouldn't it be:

#. The name of your account
#. Project name ("nice-**{$INSTALLATION}**)
#. Check
…